dts/riscv/bflb/bl61x.dtsi

```diff
@@ -89,6 +89,10 @@
 				#interrupt-cells = <2>;
 				interrupt-controller;
 			};
+
+			cache {
+				compatible = "xuantie,xtheadcmo";
```

Comment:
This should be added before the driver not together with SoC. Due to characteristics of this driver (SoC uses it) the devicetree should be added in the binding commit (before the driver).